.github/workflows/test.yml

```diff
@@ -69,26 +71,76 @@ jobs:
             docker:
               - python/**
               - docker/**
+            docker_base:
+              - docker/base/**
           treat-pr-as-push: true
 
-  set_docker_image_tag:
-    name: Set docker image tag
+  base_image:
+    name: >
```

Comment:
This is hideous, I know. But it ends up being a nice label of what base image is being used/built.

It'd be cleaner if I made "set image tag" step its own job, but I'd rather have a cleaner dag on the [test worfklow](https://github.com/NYCPlanning/data-engineering/actions/runs/9681237466) and a nice name than simplify this horrible-looking expression